sdk/build.gradle

```diff
@@ -7,12 +7,12 @@ if (rootProject.file('gradle/gradle-mvn-push.gradle').exists()) {
 }
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 26
+    buildToolsVersion '26.0.2'
 
     defaultConfig {
         minSdkVersion 19
-        targetSdkVersion 25
+        targetSdkVersion 26
```

Comment:
[q] Why did we bump this?

Reply:
`targetSdkVersion` specifies the API Level on which the application is designed to run. This should generally be (in our case) the highest API number.

Reply:
duh! whoops